packages/contracts-bedrock/contracts/universal/StandardBridge.sol

```diff
@@ -418,6 +370,11 @@ abstract contract StandardBridge {
         uint32 _minGasLimit,
         bytes memory _extraData
     ) internal {
+        require(
```

Comment:
Couldn't resist making this a little more defensive.

Reply:
Do we have test coverage that this will not break replayability?

Reply:
I believe this line will never actually be triggered in our current codebase, so maybe it's better to use `assert`. Also would not break replayability because the ETH value comes out of the `L2CrossDomainMessenger` either way. If the first message fails then the ETH is still stuck in the `L2CrossDomainMessenger` to be sent again on replay.